README.md

```diff
@@ -79,15 +79,14 @@ decisions and trade-offs.
 
 1. Clone the repository in a local folder:
   ```
-  $ git clone git@github.com:tweag/nickel.git
-  $ cd nickel
-  nickel$
+  git clone git@github.com:tweag/nickel.git
+  cd nickel
```

Comment:
Not sure whether this is an improvement. I like prefixing shell commands by `$` or `#` because it clearly separates them from the output, i.e.
```
# command
error: bla
```
is clearer than
```
command
error: bla
```

Reply:
Yeah, I initially thought it can be annoying when copy-pasting (you have to select the right part of the example), but this is not very important and I agree this is less clear. Reverted in the last commit.